Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: init test test-cov lint format
 
 CODE = flake8_pytest_style
-TEST = pytest --verbosity=2 --showlocals --strict --cov=$(CODE)
+TEST = poetry run pytest --verbosity=2 --showlocals --strict --cov=$(CODE)
```

Comment:
Please update to the latest master, this is fixed there.
In future PRs please try not to mix unrelated changes.